### ISSUE TITLE
Tuning Model

### ISSUE THREAD

**Issue by yujinjcho** [yujinjcho/movie_recommendations#4]
* add regularization
* Experiment with learning rate
* break annotations into train, validation, and test sets
* be able to measure accuracy given changes

**Comment by yujinjcho** [comment]
next steps: experiment with lambda calculation and get f-score
